pkg/NAMESPACE

```diff
@@ -65,4 +65,5 @@ export(
        "setCheckpointDir"
       )
 export("sparkR.init")
+export("print.jobj")
```

Comment:
I realize this is part of the S3 OO system that is used for printing a brief description of `jrdd` of an RDD. However do we need to export it?